src/PointwiseFunctions/GeneralRelativity/ComputeGhQuantities.hpp

```diff
@@ -456,4 +470,87 @@ tnsr::a<DataType, SpatialDim, Frame> spacetime_deriv_of_norm_of_shift(
     const tnsr::iaa<DataType, SpatialDim, Frame>& phi,
     const tnsr::aa<DataType, SpatialDim, Frame>& pi) noexcept;
 // @}
+
+namespace Tags {
+/*!
+ * \ingroup GeneralRelativityGroup
+ * \brief Compute item to get time derivative of the spatial metric from
+ *        generalized harmonic and geometric variables
+ *
+ * \details See `time_deriv_of_spatial_metric()`. Can be retrieved using
+ * `gr::Tags::SpatialMetric` wrapped in `Tags::dt`.
+ */
+template <size_t SpatialDim, typename Frame>
+struct TimeDerivSpatialMetricCompute
+    : ::Tags::dt<gr::Tags::SpatialMetric<SpatialDim, Frame, DataVector>>,
+      db::ComputeTag {
+  using argument_tags =
+      tmpl::list<gr::Tags::Lapse<DataVector>,
+                 gr::Tags::Shift<SpatialDim, Frame, DataVector>,
+                 Phi<SpatialDim, Frame>, Pi<SpatialDim, Frame>>;
+  static constexpr tnsr::ii<DataVector, SpatialDim, Frame> (*function)(
+      const Scalar<DataVector>&, const tnsr::I<DataVector, SpatialDim, Frame>&,
+      const tnsr::iaa<DataVector, SpatialDim, Frame>&,
+      const tnsr::aa<DataVector, SpatialDim, Frame>&) =
+      &time_deriv_of_spatial_metric<SpatialDim, Frame>;
+  using base =
```

Comment:
`base` and `type` are not part of the compute item interface. Please remove these.

Reply:
I've removed `type`...

... but `base` is needed when putting compute items on domain interfaces via `InterfaceComputeItem` (which inherits from `Interface<, Tag>`, which in turn inherits from `Interface<,Tag::base>`). While these compute items are not yet needed on domain interfaces, they might be in future.